module/module.go

```diff
@@ -27,15 +27,15 @@ type (
 		Version  string
 	}
 
-	// ModuleInstance represents an instance of the JS module.
-	ModuleInstance struct {
+	// JSModuleInstance represents an instance of the JS module.
+	JSModuleInstance struct {
 		mod *JSModule
 	}
 )
 
 var (
 	_ k6modules.Module   = &RootModule{}
-	_ k6modules.Instance = &ModuleInstance{}
+	_ k6modules.Instance = &JSModuleInstance{}
```

Comment:
I had to rename this to avoid the stuttering which was also causing a lint error.